idseq_pipeline/commands/accessionid2seq_functions.py

```diff
@@ -42,6 +43,59 @@ def parse_reads(annotated_fasta, db_type):
                             read2seq[ma.group(4).rstrip()] = [sequence.rstrip(), ma.group(1), ma.group(2), ma.group(3)]
     return read2seq
 
+def compress_coverage(coverage):
+    keys = sorted(coverage.keys())
+    if len(keys) <= 1:
+        return coverage
+    output = {}
+
+    start = keys[0]
+    current = start
+    val = coverage[start]
+
+    for k in keys[1:]:
+        if (k-current) == 1 and coverage[k] == val:
+            current = k
+        else:
+            output["%d-%d" % (start, current)] = val
```

Comment:
I presume these are half-open `[start, current)` intervals.   Worth making sure this is assumed everywhere else.